jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -30,28 +30,35 @@ CircleCI test splitting lets you intelligently define where splits happen across
 
 It is also possible to use the CLI to [manually allocate tests](#manual-allocation) across parallel environments. Another alternative is to use [environment variables](#using-environment-variables-to-split-tests) instead of the CLI to configure split tests.
 
+## Quickstart
+{: #quickstart }
+
+The following how-to guide walks you through the steps required to glob your test files, split your tests evenly across parallel containers or VMs, and then run your tests in parallel:
+
+* **How-to**: [Use the CircleCI CLI to split tests](/docs/use-the-circleci-cli-to-split-tests/)
+
 ## Specify a job's parallelism level
```

Comment:
Yeah good point 😭 I'll update this

Reply:
I'll just use the go example from further down but remove the split-by and timings-type flags to simplify, use defaults, and have it show split by name?

Reply:
i prefer to have the default in the docs show the split-by=timings because that's what we want customers to use.  and im worried if we dont include that in by default, ppl will miss it and just rely on the split by name